docs/fastdds/transport/tcp/tcp.rst

```diff
@@ -31,91 +34,129 @@ TCPTransportDescriptor
-| ``logical_port_increment``   | ``uint16_t``         | 2       | Increment between logical ports to try               |
-|                              |                      |         | during RTCP negotiation.                             |
-+------------------------------+----------------------+---------+------------------------------------------------------+
-| ``enable_tcp_nodelay``       | ``bool``             | false   | Enables the TCP_NODELAY socket option.               |
-+------------------------------+----------------------+---------+------------------------------------------------------+
-| ``calculate_crc``            | ``bool``             | true    | True to calculate and send CRC on message headers.   |
-+------------------------------+----------------------+---------+------------------------------------------------------+
-| ``check_crc``                | ``bool``             | true    | True to check the CRC of incoming message headers.   |
-+------------------------------+----------------------+---------+------------------------------------------------------+
-| ``apply_security``           | ``bool``             | false   | True to use TLS. See |TLSconfig|.                    |
-+------------------------------+----------------------+---------+------------------------------------------------------+
-| ``tls_config``               | ``TLSConfig``        |         | Configuration for TLS. See |TLSconfig|.              |
-+------------------------------+----------------------+---------+------------------------------------------------------+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Member
+    - Data type
+    - Default
+    - Description
+  * - |SocketTransportDescriptor::sendBufferSize-api|
+    - ``uint32_t``
+    - ``0``
+    - Size of the sending buffer of the socket (octets).
+  * - |SocketTransportDescriptor::receiveBufferSize-api|
+    - ``uint32_t``
+    - ``0``
+    - Size of the receiving buffer of the socket (octets).
+  * - |SocketTransportDescriptor::interfaceWhiteList-api|
+    - ``vector<string>``
+    - empty
+    - List of allowed interfaces
+      See |InterfaceWhitelist|.
+  * - |SocketTransportDescriptor::TTL-api|
+    - ``uint8_t``
+    - ``1``
+    - Time to live, in number of hops.
+  * - |TCPTransportDescriptor::listening_ports-api|
+    - ``vector<uint16_t>``
+    - empty
+    - List of ports to listen as *server*.
+  * - |TCPTransportDescriptor::keep_alive_frequency_ms-api|
+    - ``uint32_t``
+    - 5000
+    - Frequency of RTCP keep alive requests (in ms).
+  * - |TCPTransportDescriptor::keep_alive_timeout_ms-api|
+    - ``uint32_t``
+    - 15000
+    - Time since sending the last keep alive request to consider a connection as broken (in ms).
+  * - |TCPTransportDescriptor::max_logical_port-api|
+    - ``uint16_t``
+    - 100
+    - Maximum number of logical ports to try during RTCP negotiation.
+  * - |TCPTransportDescriptor::logical_port_range-api|
+    - ``uint16_t``
+    - 20
```

Comment:
You are completely correct. I just moved the table but I did not realize about this. :sweat_smile: It would be nice to have a common style applicable to all the documentation. I am looking into the QoS policies and there the types are not emphasized nor are the default values (with the exception to ``true`` and ``false``). Maybe I can use this style and from here onward enforce it.